src/main/extension/extension_install.cpp

```diff
@@ -45,7 +45,9 @@ const vector<string> ExtensionHelper::PathComponents() {
 
 string ExtensionHelper::ExtensionDirectory(DBConfig &config, FileSystem &fs) {
 #ifdef WASM_LOADABLE_EXTENSIONS
-	static_assertion(0, "ExtensionDirectory functionality is not supported in duckdb-wasm");
+#ExtensionDirectory functionality is not supported in duckdb - wasm
```

Comment:
Should this not stay as a static assert? - looks like this was changed accidentally